packages/debug/src/browser/debug-session.tsx

```diff
@@ -74,7 +83,11 @@ export class DebugSession implements CompositeTreeElement {
     protected readonly childSessions = new Map<string, DebugSession>();
     protected readonly toDispose = new DisposableCollection();
 
-    private isStopping: boolean = false;
+    protected isStopping: boolean = false;
+    /**
+     * Number of millis after a `stop` request times out.
+     */
+    protected stopTimeout = 5_000;
```

Comment:
This just doesn't feel like a constant that should be different for different _products_. If it varies, I would argue it should be either a user setting (slow machine) or different by debug adapter (slow adapter). Also: how would downstream adopters customize this value? Wouldn't a constructor parameter be more appropriate?

Reply:
Good idea. I moved it to the ctor.

https://github.com/eclipse-theia/theia/pull/11984/commits/7d995566abc9c5165bc22a15955fb9854b2c04e8